cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -121,7 +121,7 @@ private boolean setRegulatingControl(CgmesRegulatingControlForStaticVarCompensat
             okSet = true;
         } else {
             context.fixed("SVCControlMode", () -> String.format("Invalid control mode for static var compensator %s. Regulating control is disabled", svc.getId()));
-            regulationMode = StaticVarCompensator.RegulationMode.OFF;
+            regulationMode = null;
```

Comment:
```suggestion
            regulationMode = StaticVarCompensator.RegulationMode.VOLTAGE;
             svc.setRegulating(false);
```
To be discussed but here we need to avoid having a null RegulationMode + disable the regulation?

Reply:
maybe we previously need to check that the `svc.getRegulationMode()` is not null  and if so set it as `VOLTAGE` to avoid null?

Reply:
Same opinion for me, this is probably better to avoid the null management and in such a case, we just set the mode to most used one (VOLTAGE). 
But if we really want to support the "no mode" setting when regulating is false, we need to change to API to manager an Optional<RegulationMode>> to avoid nullable mode. Maybe more complex than just set a whatever value when regulating is false.